opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -218,6 +222,10 @@ namespace Opm
             // Increment timer, remember well state.
             ++timer;
             prev_well_state = well_state;
+            // Compute Well potentials (only used to determine default guide rates for group controlled wells)
+            if (schedule->numGroups() > 0 ) {
```

Comment:
Will there not always be at least one group (the FIELD group)? We can still in theory have controls on that group so checking that we have > 1 group would not be good either. There must be some other, not-too-hard way to ask if we need to compute the potentials.